test/core/ticktree/TickTreeTest.t.sol

```diff
@@ -25,67 +25,67 @@ import {AbstractMangrove, TestTaker, MangroveTest, IMaker, TestMaker} from "mgv_
 import "mgv_src/MgvLib.sol";
 import "mgv_lib/Debug.sol";
 
-// Base class for test of Mangrove's tick tree data structure
+// Base class for test of Mangrove's bin tree data structure
```

Comment:
As remarked further up, it's not quite consistent whether we call it "tick tree" or "bin tree", eg the files here are still called `TickTree*` but the comments mention "bin trees".

Reply:
Moved to tick tree in 8f1399bb7ac80f7ddc9ff4a6e6311179cc9f41d7